packages/eslint-plugin/lib/rules/no-extra-spacing-attrs.js

```diff
@@ -69,6 +75,14 @@ module.exports = {
               return fixer.removeRange([current.range[1] + 1, after.range[0]]);
             },
           });
+        } else if (disallowMissing && spacesBetween < 1) {
+          context.report({
+            loc: NodeUtils.getLocBetween(current, after),
```

Comment:
```suggestion
            loc: after.loc,
```
I think the loc should be changed because there is no space between the two nodes, so users cannot find the error on their editor. (red error line)

- `NodeUtils.getLocBetween(current, after)` version
<img width="835" alt="스크린샷 2023-01-18 오후 12 39 26" src="https://user-images.githubusercontent.com/41323220/213078417-449bb87a-107b-4887-bd65-68eaf95a58fb.png">

- `after.loc` version
<img width="993" alt="스크린샷 2023-01-18 오후 12 40 35" src="https://user-images.githubusercontent.com/41323220/213078753-fefdb010-2427-4942-81c0-08a63839c43e.png">

Reply:
that makes sense. I've applied the suggestion and also updated the docs. please take a look. thanks.